lib/perl/Genome/Model/Build/SomaticValidation.pm

```diff
@@ -319,4 +321,48 @@ sub get_feature_list_from_reference {
     return $self->reference_sequence_build->get_feature_list($feature_list_accessor);
 }
 
+sub individual {
+    my $self = shift;
+    return $self->subject;
```

Comment:
Taxon is rarely used and when used is often for strange cases of combining samples from different individuals.  I suggest failing with an appropriate error message rather than supporting SomVal builds with taxon subjects.  ClinSeq is intended for supporting individual-based analysis and taxon is outside the scope.